src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs

```diff
@@ -32,59 +33,85 @@
 
 
 (react/defc ImportWorkspaceMethodsConfigurationsList
-  {:render
-   (fn [{:keys [props]}]
+  {:component-did-mount
+   (fn [{:keys [state props]}]
+     (utils/ajax-orch
+       (str "/workspaces/" (:selected-workspace-namespace props) "/" (:selected-workspace props) "/methodconfigs")
+       {:on-done (fn [{:keys [success? xhr]}]
+                   (if success?
+                     ((fn []
```

Comment:
This is just creating a function and calling it. If you want to execute multiple statements, use `do`:
```
(if success?
  (do
    (swap! state...)
```